Fix recipes when rerendered

mcbarton opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

mcbarton committed:
Bump build number

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

mcbarton commented:
@conda-forge-admin, please rerender

mcbarton commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.15.11.56.5

mcbarton committed:
Remove added jobs

mcbarton commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18593390220) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18593390220](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18593390220). Examine the logs at this URL for more detail.</sub>

mcbarton committed:
Refine clangdev version skip condition

Updated the skip condition for clangdev versions in the build section.

mcbarton commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.15.11.56.5

mcbarton committed:
Fix osx x86 conda runner error for MacOS 15

mcbarton commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18594634629](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18594634629). Examine the logs at this URL for more detail.</sub>

mcbarton committed:
Remove broken osx jobs due to lack of float16 support

mcbarton commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18596479787) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18596479787](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18596479787). Examine the logs at this URL for more detail.</sub>

mcbarton committed:
Fix meta.yaml

mcbarton commented:
@conda-forge-admin, please rerender

mcbarton commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.15.11.56.5

mcbarton committed:
Try to fix osx x86 tests on conda runners

mcbarton committed:
Fix ln command from previous commit

mcbarton committed:
Try fix

mcbarton committed:
Try fix

mcbarton committed:
Update run_test.sh

mcbarton committed:
Debug

mcbarton committed:
Try again to fix

mcbarton committed:
Debug

Updated the run_test.sh script to include symbolic link creation and removal for libLTO.dylib, and modified the clang++ command for testing.

mcbarton committed:
Refactor run_test.sh to remove libLTO link creation

Removed symbolic link creation for libLTO.dylib and added cleanup for libLTO.$clangdev.dylib.

mcbarton committed:
Debug

mcbarton committed:
Try fix

Removed LTO library checks and updated clang command.

mcbarton committed:
Add verbose flag to clang++ command in run_test.sh

mcbarton committed:
Update run_test.sh

mcbarton committed:
Fix linking to LTO library in run_test.sh

mcbarton commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18621030842](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18621030842). Examine the logs at this URL for more detail.</sub>

mcbarton committed:
Fix linking issue for OSX in run_test.sh

Added conditional linking for OSX to avoid libLTO errors.

mcbarton committed:
Add verbose flag to clang++ command in run_test.sh

mcbarton committed:
Update run_test.sh

mcbarton committed:
Update run_test.sh to link directly to libLTO.dylib

Remove symlink creation for libLTO and directly link to libLTO.dylib.

mcbarton committed:
Update run_test.sh

mcbarton committed:
Update run_test.sh

mcbarton committed:
Debug
